wcivf/apps/elections/templates/elections/includes/_person_card.html

```diff
@@ -4,10 +4,14 @@
 <li class="ds-candidate" style="list-style-type: none;">
   <div class="ds-candidate-body ds-stack-smaller">
     <h3 class="ds-candidate-name ds-h5">
-      <a href="{{ person_post.person.get_absolute_url }}" class="ds-card-link">{{ person_post.person.name}}</a>
+      {% if person_post.list_position %}
+        {{ person_post.list_position }}. 
+      {% endif %}
+      <a href="{{ person_post.person.get_absolute_url }}" class="ds-card-link">
+        {{ person_post.person.name}}
+      </a>
     </h3>
-    {% if person_post.elected %}🌟 <span class="elected_text">Elected:</span>
-    {% endif %}
+    {% if person_post.elected %}<span>🌟Elected🌟</span>{% endif %}
```

Comment:
I know we don't do this everywhere at the moment, but when we use emoji decoration like this, can we wrap it in `<span aria-hidden="true">`?